src/main/app/src/constants/Navbar.js

```diff
@@ -1,44 +1,28 @@
 import React from 'react';
-import {GoogleLogin, GoogleLogout} from 'react-google-login'
+import { NavLink } from 'react-router-dom';
+
```

Comment:
maybe remove these extra lines to make the top section more concise. Not necessary.